valo/vaadin-date-picker.html

```diff
@@ -0,0 +1,172 @@
+<link rel="import" href="color.html">
+<link rel="import" href="sizing-and-spacing.html">
+<link rel="import" href="style.html">
+<link rel="import" href="typography.html">
+<link rel="import" href="vaadin-text-field.html">
+<!--<link rel="import" href="vaadin-button.html">-->
+<link href="https://fonts.googleapis.com/css?family=Cabin" rel="stylesheet">
+
+
+<dom-module id="date-picker-styles" theme-for="vaadin-date-picker">
+  <template>
+    <style include="vaadin-date-picker-default-theme">
+      :host {
+        font-family: 'Cabin', sans-serif;
+        width: 250px;
+        max-width: 100%;
+      }
+
+      [part="text-field"] {
+        max-width: 100%;
+      }
+    </style>
+  </template>
+</dom-module>
+
+<dom-module id="month-calendar-styles" theme-for="vaadin-month-calendar">
+  <template>
+    <style include="vaadin-month-calendar-default-theme">
+      [part~="date"] {
+        padding: 10px 0;
+        font-size: 16px;
+        height: 42px;
+        width: 42px;
```

Comment:
Alot of magic numbers which should be replaced with computed values or variables.